demo/app/lib/views/wallet_initiated_connect.dart

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import 'dart:convert';
+import 'dart:developer';
+import 'package:app/models/connect_issuer_config.dart';
+import 'package:app/models/connect_issuer_config_value.dart';
+import 'package:app/services/config_service.dart';
+import 'package:app/views/supported_credentials_list.dart';
+import 'package:app/widgets/primary_button.dart';
+import 'package:flutter/material.dart';
+import 'package:app/widgets/common_title_appbar.dart';
+import 'package:flutter/services.dart';
+import 'package:app/wallet_sdk/wallet_sdk_mobile.dart';
+
+class IssuerList extends StatefulWidget {
+  const IssuerList({Key? key}) : super(key: key);
+
+  @override
+  State<IssuerList> createState() => IssuerListState();
+}
+
+class IssuerListState extends State<IssuerList> {
+  List<ConnectIssuerConfig> connectIssuerConfigList = List.empty(growable: true);
+  late final String redirectURI;
```

Comment:
Currently we put redirectURI into ConnectIssuerConfig>, so it not need here

Reply:
done removed